src/application_protocols/dubbo/dubbo_hessian2_serializer_impl.cc

```diff
@@ -108,9 +108,9 @@ DubboHessian2SerializerImpl::deserializeRpcResult(Buffer::Instance& buffer,
     result->setException(true);
     break;
   case RpcResponseType::ResponseWithNullValue:
-  case RpcResponseType::ResponseNullValueWithAttachments:
     has_value = false;
```

Comment:
Should we change the name of variable ```has_value```  to ```has_value_or_attachment```?

Reply:
Yes, I think has_value_or_attachment is more accurate.